contracts/src/v0.8/dev/VRFSubscriptionBalanceMonitor.sol

```diff
@@ -0,0 +1,314 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.6;
+
+import "../ConfirmedOwner.sol";
+import "../interfaces/KeeperCompatibleInterface.sol";
+import "../interfaces/VRFCoordinatorV2Interface.sol";
+import "../interfaces/LinkTokenInterface.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+
+/**
+ * @title The VRFSubscriptionBalanceMonitor contract.
+ * @notice A keeper-compatible contract that monitors and funds VRF subscriptions.
+ */
+contract VRFSubscriptionBalanceMonitor is ConfirmedOwner, Pausable, KeeperCompatibleInterface {
```

Comment:
`ConfirmedOwner` includes ownership transfer capabilities right?

Reply:
yep! Transfer and then approve.